<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cqp::ConnectDialog</class>
 <widget class="QDialog" name="cqp::ConnectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="addressLbl">
     <property name="text">
      <string>Address</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="address">
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="typeLbl">
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QRadioButton" name="typeSite">
     <property name="text">
      <string>Site Agent</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/icons/site</normaloff>:/icons/site</iconset>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">serviceGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QRadioButton" name="typeDevice">
     <property name="text">
      <string>Device</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/icons/device</normaloff>:/icons/device</iconset>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">serviceGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QRadioButton" name="typeManager">
     <property name="text">
      <string>Manager</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/icons/manager</normaloff>:/icons/manager</iconset>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">serviceGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>cqp::ConnectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>cqp::ConnectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="serviceGroup"/>
 </buttongroups>
</ui>
